models/state.coffee

```diff
@@ -6,13 +6,15 @@ module.exports = class State extends Backbone.Model
   defaults:
     view_mode: 'grid'
     lightbox: false
+    sort: 'updated_at'
 
   initialize: ->
     # set values from cookies
     each keys(@defaults), (key) =>
       @set(key, val) if val = cookies.get key
 
     @on 'change:view_mode', @setCookie
+    @on 'change:sort', @setCookie
 
   setCookie: (model, value)->
     cookies.set keys(model.changed)[0], value
```

Comment:
This is fine, but this model should probably be named something more specific like 'ViewState' or 'UIState'